conda/README.md

```diff
@@ -0,0 +1,192 @@
+
+A Conda package could contain one or more Zowe Plugins, and a Conda package could contain non-Zowe code alongside Zowe Plugins.
+This is left up to the program vendor, and regardless the install process is the same:
+
+`conda install package_name`
+
+If the Zowe environment variables are set, such a package may automatically register plugins into the Zowe instance of your choice.
+
+### Zowe Plugin Configuration
+
+Aside from possible automation during install and uninstall, Conda does not manage Zowe, its configuration, or configuration of the Plugins.
+However, Conda does manage the package files, and therefore you can do additional Zowe tasks on the Plugins by going into the Conda environment.
+Zowe Plugins are intended to be found in a standardized location in the Conda environment,
+
+`/opt/zowe/plugins`
+
+This folder contains plugins, which in turn contain sub-folders that are the Zowe components that they utilize.
+If a plugin uses multiple Zowe components, its contents could be found within multiple component folders.
+
+`/opt/zowe/plugins/my_plugin/app-server`
+`/opt/zowe/plugins/my_plugin/cli`
+
+### Zowe Package Structure
+
+Zowe Plugins packaged into Conda follow the structure outlined here: https://github.com/zowe/zowe-install-packaging/issues/1569
+This structure allows for plugin to have content meant for one or more Zowe components.
+The Conda packages extend this by allowing for more than one Plugin, or a mix of Zowe Plugins and other software to be within a single package.
+
+## Building Conda Packages for Zowe
+
+This document is intended to be provided with example scripts by the Zowe community, which shows you how you can build a simple Zowe plugin into a Conda package.
+This is not intended to be a one-size-fits-all set of scripts. If you have more advanced needs, you can use these scripts as a basis for writing your own scripts.
+
+To make a Conda package, you need conda-build, which you can install into a Conda environment:
+
+`conda install conda-build`
+
+Once you have it, you can build a package via
+
+`conda build path/to/build/scripts`
+
+However, first you must set up the build information.
+
+### Defining package properties
+Conda needs a metadata file, `meta.yaml` to state information about the package, such as dependencies, what OS it supports, its name and version.
+This information can be programmatically found, and Zowe provides examples of how to do this by reading Zowe's own metadata files into this one.
+
+### Creating Build Step
+It's recommended not to build your code from scratch to put into Conda.
+Rather, build your code however you want, and then just copy the contents into a conda package. This keeps the Conda scripting small and simple.
+
+In the same folder as `meta.yaml`, Conda requires `build.sh` for building on Unix, Linux, or z/OS and `build.bat` for Windows.
+Except for z/OS, this script does not determine where your package can be used, it's just about where you are building it.
```

Comment:
Can you reword this sentence? I'm absolutely dense, but it took me like 7 times to re-read it

"The script determines not where the package can be used, but where it can be built. z/OS is the exception because building on z/OS preserves tagging information. So, if you tag your files incorrectly, the user will have to sort out encoding issues on their own. This is why a z/OS package has to be marked as 'zos-z' content below.  "